File: Codigo_Reescrito/Codigo_Python/ChromeWindow1.py
"""
    Processo chrome aberto independentemente para permitir "burlar" o GIL do Python e utilizar de multiprocesamento 
        para acelerar a pesquisa.
    
"""

import Config                                           # Informações básicas

from selenium import webdriver as Swd

from Funcoes import esperar                             # Substituí o time.sleep

from Config import SteamTags                            #   apenas para facilitar o acesso

from WebScraping import lerPagina                       # Irá devolver uma lista de jogos        

from Funcoes import escreverListaTemp                   # Escreve o arquivo temporário da lista de jogos













nomeInstancia = Config.instanciaUm

driver = Swd.Chrome(executable_path=Config.chromeDriver)

linkDaPagina = SteamTags.linkComTag(SteamTags.online_coop)

driver.get(linkDaPagina)
    
listaBonsJogos = lerPagina(driver)

escreverListaTemp(listaBonsJogos, nomeInstancia)

File: Codigo_Reescrito/Codigo_PHP/ChromeWindow1.php
<?php

/*
    Processo chrome aberto independentemente para permitir "burlar" o GIL do Python e utilizar de multiprocesamento 
        para acelerar a pesquisa.
    
*/


require_once 'Config.php';

require_once 'Dependencias/vendor/autoload.php';
use Facebook\WebDriver\Remote\RemoteWebDriver;
use Facebook\WebDriver\Remote\DesiredCapabilities;

require_once 'Classes/SteamTags.php';
require_once 'Funcoes/lerPagina.php';
require_once 'Funcoes/escreverListaTemporaria.php';

$nomeInstancia = $instanciaUm;

$serverUrl = 'http://localhost:' . $chromeDriverPort;
$webdriver = RemoteWebDriver::create($serverUrl, DesiredCapabilities::chrome());

$linkDaPagina = new SteamTags();
$linkDaPagina = $linkDaPagina->linkComTag($linkDaPagina->online_coop, $valorMaximo);

$webdriver->get($linkDaPagina);

$listaBonsJogos = lerPagina($webdriver);

escreverListaTemporaria($listaBonsJogos, $nomeInstancia);
